proto/internal/temporal/server/api/persistence/v1/namespaces.proto

```diff
@@ -59,6 +59,7 @@ message NamespaceConfig {
     string history_archival_uri = 5;
     temporal.api.enums.v1.ArchivalState visibility_archival_state = 6;
     string visibility_archival_uri = 7;
+    map<string, string> custom_search_attributes_alias = 8;
```

Comment:
Shouldn't it be
```suggestion
    map<string, string> custom_search_attributes_aliases = 8;
```
?

Reply:
Add a comment saying that it is field to alias map and inverse map is built on read.